### DISCUSSION TITLE
Ctrl+H/J cannot be undone in the editor

### DISCUSSION THREAD

**Discussion by cihe13375** [ppy/osu#24063] [Q&A] [answered]
How to reproduce:
1) open a map in the editor
2) select an object
3) Ctrl+H or Ctrl+J
4) Ctrl+Z
expected: the Ctrl+H/J operation in 3) is undone
actual: nothing happens. If there are extra operations before Ctrl+H/J, they are undone instead.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/24065
